c++/include/orc/Writer.hh

```diff
@@ -262,6 +262,19 @@ namespace orc {
      * @return if not set, the default is false
      */
     bool getUseTightNumericVector() const;
+
+    /**
+     * Set the initial buffer capacity of output stream.
+     * Each column contains multiple output streams with buffers,
```

Comment:
Class CompressionStream is a subclass of class BufferedOutputStream which contains the buffer. And BufferedOutputStream has no logic associated with the compression algorithm, so it may not be appropriate to use compressed stream to describe it.

Reply:
Then it makes sense to me to use `BufferedOutputStream` explicitly. We have defined several output streams (e.g. one from `OrcFile.hh`). It should be clear to user which one we are referring to.